mongoengine/document.py

```diff
@@ -463,6 +463,12 @@ def delete(self, **write_concern):
         """
         signals.pre_delete.send(self.__class__, document=self)
 
+        #Delete FileFields seperately 
```

Comment:
Please add a space after the '#' here and in the test, and also correct seperately to separately :-)